src/Workspace/index.jsx

```diff
@@ -370,6 +372,14 @@ class Workspace extends React.Component {
         if (this.workspaceStates.includes(data.status)) {
           const workspaceLaunchStepsConfig = this.getWorkspaceLaunchSteps(data);
           let workspaceStatus = data.status;
+          if (this.state.initiatePayModelCallUponTermination && workspaceStatus === 'Not Found') {
+            this.state.initiatePayModelCallUponTermination = false;
+            this.getWorkspacePayModel().then((payModelData) => {
+              this.setState({
+                payModel: payModelData,
```

Comment:
just curious when resetting paymodels, this will be reset to vwhat value? `{}` or something else? sry I didn't check the hatchery codes